projects/storefrontapp-e2e-cypress/cypress/integration/regression/cpq-configuration.e2e-spec.ts

```diff
@@ -116,4 +132,75 @@ context('CPQ Configuration', () => {
       );
     });
   });
+
+  const waitForConfiguratorLoad = () => {
```

Comment:
Could you move this method to the helper class and rename it?
The current method name is not correct. This methos waits until the **product card** is rendered accordingly.
The test class should only contain tests